app/controllers/api/verify/document_capture_controller.rb

```diff
@@ -0,0 +1,80 @@
+module Api
+  module Verify
+    class DocumentCaptureController < BaseController
+      self.required_step = 'document_capture'
+      include ApplicationHelper
+      include EffectiveUser
```

Comment:
now that this is in the base controller, this is redundant

```suggestion
```

Reply:
I had suggested to include it here even if redundant to make a bit more explicit, since the fact it's included in the base controller is more incidental and could change over time.

Case in point: I would like to move [this test out of the base controller](https://github.com/18F/identity-idp/pull/6483/files#diff-a621e837de21e82152931bd65ad3d5c47b26dd8fa8b4b3d492f3f7abc20fc4faR24).

I don't feel overly strongly one way or the other, though.